sr_unix_gnp/cmu_getclb.c

```diff
@@ -49,10 +49,12 @@ struct CLB *cmu_getclb(cmi_descriptor *node, cmi_descriptor *task)
 			if (0 == memcpy(ai_ptr->ai_addr, (sockaddr_ptr)(&p->peer_sas), ai_ptr->ai_addrlen))
 			{
 				SIGPROCMASK(SIG_SETMASK, &oset, NULL, rc);
+				freeaddrinfo(ai_ptr);
```

Comment:
The commit comment incorrectly identifies the name of this module as cmj_getclb.c. Should be cmu_getclb.c

Reply:
Fixed.